### ISSUE TITLE
how to use online sync on linux

### ISSUE THREAD

**Issue by lbngoc** [utkarsh9891/PackageSync#5]
I want to sync my ST installed packages and settings between my laptop (linux) and pc (mac).
How can I do it ? I enable online syncing and set online sync folder="Shared" (a folder on my Google drive), then I select command "Push Settings to Online Sync Folder" but nothing is happened...
I guess that I need to install a sync client tool on my computer, but I can not find it. Google drive does not have a sync client on linux.

**Comment by utkarsh9891** [comment]
You would have to choose an alternative sync client that works on both OS X and Linux. FreeFileSync is one such client. Alternatively, you could also look at the post [here](http://www.howtogeek.com/196635/an-official-google-drive-for-linux-is-here-sort-of-maybe-this-is-all-well-ever-get/) on how to setup Google Drive on Linux. Do note that I haven't tried this myself.

**Comment by lbngoc** [comment]
I see that in many other packages, they have use gist.github.com for store data and sync. I think it will help users easily config and use.
@utkarsh9891: could you add this feature to new version of this package ? It will much awesome !!!

**Comment by utkarsh9891** [comment]
Seems like a nice idea. Would look into the feasibility for this & update the package accordingly. But, unfortunately, I would require some time before I start working on this. You would have to resort to some other file sync client till then.
Or even better if you could add a fork for the gist usage. 👍

**Comment by utkarsh9891** [comment]
@lbngoc: Sorry for the extremely late update but unfortunately I have been unable to find time to work on this package. I did do a feasibility study of the use of gists for syncing packages but the effort required did not seem worth the output.  
Having said that, I would definitely work towards adding this feature in a future version. Meanwhile, I would advise you to try syncing via [DropBox](https://www.dropbox.com/install-linux) till then.

Closing this thread as there is no action to be taken at this point of time regarding the feature update.
